felix/config/param_types.go

```diff
@@ -554,28 +554,62 @@ func (r *RegionParam) Parse(raw string) (result interface{}, err error) {
 	return raw, nil
 }
 
-type RouteTableRangeParam struct {
+// reserved linux kernel routing tables (cannot be targeted by routeTableRanges)
+var routeTablesReservedLinux = []int{253, 254, 255}
```

Comment:
According to [this](http://linux-ip.net/html/routing-tables.html) it might also be wise to reserve table 0, though I'm not sure if other logic already excludes 0.

Reply:
0 is the empty (unspecified) val I believe? I assumed that value gets parsed as "none" , but will dig into it and confirm :slightly_smiling_face: